src/main/java/com/derongan/minecraft/mineinabyss/player/PlayerListener.kt

```diff
@@ -73,4 +80,19 @@ object PlayerListener : Listener {
         player.playerData.addExp(amount.toDouble())
     }
 
+    @EventHandler
+    fun EntityPotionEffectEvent.onPlayerHit() {
+        if (entity is Player) {
+            if (cause == EntityPotionEffectEvent.Cause.MILK) {
+                isCancelled = true
+                (entity as Player).error("${ChatColor.BOLD}Milk ${ChatColor.RED}has been disabled")
+            }
+            else if (cause != EntityPotionEffectEvent.Cause.COMMAND) {
+                if (newEffect?.type == PotionEffectType.DAMAGE_RESISTANCE) {
```

Comment:
I don't like that this this disables all damage resistance potions. Still think the better option is to remove the crafting recipe.

Reply:
I mean resistance = 180 block drops basically so I dont see why we wouldnt get rid of every single version of it.
Be that tipped arrows, splash potions, lingering potions. Made it only work via commands as we do kind of need it for event bosses.
Disabling the recipe could work but it doesn't mean that it wont be obtainable.